TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/bungeecord/listeners/ServerKick.java

```diff
@@ -17,6 +18,9 @@ public void onServerKickEvent(ServerKickEvent event) {
             TimoCloudBungee.getInstance().info("No fallback server found");
             return;
         }
+        TimoCloudBungee.getInstance().info(event.getCancelServer().getName() + " kicked player " + event.getPlayer().getName() + " because of: " + TextComponent.toLegacyText(event.getKickReasonComponent()));
```

Comment:
Do server owners always want to have a log of the kicked players in their console?